packages/hs-react-ui/src/components/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, SyntheticEvent } from 'react';
+import { text, select} from '@storybook/addon-knobs';
+import TextInput from './TextInput';
+import { TextInputTypes } from '../../enums/TextInputTypes';
+import { mdiComment } from '@mdi/js';
+import { action } from '@storybook/addon-actions';
+import Icon from '@mdi/react';
+import * as IconPaths from '@mdi/js';
+
+export default {
+  title: 'TextInput',
+  component: TextInput,
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/3r2G00brulOwr9j7F6JF59/Generic-UI-Style?node-id=102%3A29'
+  }
+};
+
+export const Basic = () => {
+
+  const [inputValue, setInputValue] = useState('');
+
+  const options = {
+    none: '',
+    ...IconPaths
+  };
+
+  const getIconPath = (path: string) => path ? <Icon size={'16px'} path={path} /> : null
+  
+  const isMultiLine = (inputValue: string) => (inputValue.length > 15) ? true : false;
+
+  const isError = (inputValue: string) => (inputValue.length < 5 && inputValue.length > 0) ? 'Short Message Error' : null;
+
+  return (
+
```

Comment:
Could condense these lines a bit
```suggestion
  const [inputValue, setInputValue] = useState('');
  const options = {
    none: '',
    ...IconPaths
  };
  const getIconPath = (path: string) => path ? <Icon size={'16px'} path={path} /> : null
  const isMultiLine = (inputValue: string) => (inputValue.length > 15) ? true : false;
  const isError = (inputValue: string) => (inputValue.length < 5 && inputValue.length > 0) ? 'Short Message Error' : null;
  
  return (
```

Reply:
Resolved